### ISSUE TITLE
webp_auto_support: false does not work

### ISSUE THREAD

**Issue by NiklasBr** [pimcore/pimcore#7357]
### Expected behavior

According to the documentation [it should be possible to disable](https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_WebP-Support) to disable WebP. Deleting the temproary files, and then running the `bin/console pimcore:thumbnails:image` command should not generate any WebP files.

### Actual behavior

WebP images are generated, both on disk and in the HTML markup.

### Steps to reproduce

Pimcore 6.8.2
PHP 7.4.11
Imagemagick 7.0.10-34

Update the config, clear the cache, delete the temporary files. Re-process all assets.

### Code example
File pimcore/app/config/config.yml:
```yml
pimcore:
    assets:
        image:
            thumbnails:
                webp_auto_support: false
```

File pimcore/var/config/image-thumbnails.php:
```php
return [
//...
    "detail" => [
        "items" => [
            [
                "method" => "cover",
                "arguments" => [
                    "width" => 540,
                    "height" => 700,
                    "positioning" => "topcenter",
                    "forceResize" => FALSE
                ]
            ]
        ],
        "medias" => [

        ],
        "name" => "detail",
        "description" => "",
        "group" => "Product",
        "format" => "SOURCE",
        "quality" => 85,
        "highResolution" => 0.0,
        "preserveColor" => FALSE,
        "preserveMetaData" => TRUE,
        "rasterizeSVG" => FALSE,
        "downloadable" => TRUE,
        "modificationDate" => 1603118890,
        "creationDate" => 1590420641,
        "forcePictureTag" => FALSE,
        "id" => "detail"
    ],
//...
```

Example output:
```bash
$ ls -l pimcore/web/var/tmp/image-thumbnails/assortment/16060/*
pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__detail:
-rw-rw-r-- 1 niklas niklas 27600 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas 17052 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 61545 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas 15738 Oct 19 19:26 '100577@2x.webp'

pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__hero:
-rw-rw-r-- 1 niklas niklas  70416 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas  39778 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 104289 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas  26342 Oct 19 19:26 '100577@2x.webp'

pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__list:
-rw-rw-r-- 1 niklas niklas 1781 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas 1118 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 3680 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas 2478 Oct 19 19:26 '100577@2x.webp'

pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__mini:
-rw-rw-r-- 1 niklas niklas 2556 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas 1640 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 5644 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas 3700 Oct 19 19:26 '100577@2x.webp'

pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__nav_icon:
-rw-rw-r-- 1 niklas niklas 1569 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas  886 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 3067 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas 2046 Oct 19 19:26 '100577@2x.webp'

pimcore/web/var/tmp/image-thumbnails/assortment/16060/image-thumb__3811__page:
-rw-rw-r-- 1 niklas niklas  7303 Oct 19 19:26  100577.jpg
-rw-rw-r-- 1 niklas niklas  4942 Oct 19 19:26  100577.webp
-rw-rw-r-- 1 niklas niklas 19180 Oct 19 19:26 '100577@2x.jpg'
-rw-rw-r-- 1 niklas niklas 12012 Oct 19 19:26 '100577@2x.webp'
```

**Comment by NiklasBr** [comment]
Turns out Pimcore choose to ignore it when running the command:
https://github.com/pimcore/pimcore/blob/fd693eaa2bd688b2389afe8f1a1f2d724f6ff78e/bundles/CoreBundle/Command/ThumbnailsImageCommand.php#L191-L195

**Comment by dvesh3** [comment]
Fixed by #7358
